src/btree/bt_cursor.c

```diff
@@ -1879,6 +1852,8 @@ __wt_btcur_close(WT_CURSOR_BTREE *cbt, bool lowlevel)
     if (!lowlevel)
         ret = __cursor_reset(cbt);
 
+    __wt_buf_free(session, &cbt->modify_update.buf);
+    __wt_buf_free(session, &cbt->upd_value.buf);
```

Comment:
Alex, there are now two buffers in `WT_CURSOR_BTREE *` that do the `cbt->tmp = &cbt->_tmp` on initialization so that all future references don't need `&`, and two that don't. Maybe it should be consistent? (I honestly don't care which way we go, or if you think it's fine the way it is.)

Reply:
I'm not a fan of that pattern because it obscures the fact that the `WT_ITEM` is part of the `WT_CURSOR_BTREE *` itself and not just a pointer into space. I don't feel strongly though so if anyone wants this, I'm happy to do it.

Reply:
I'm good with going one way or the other, I just don't really like mixing-and-matching.

Reply:
Fair enough. I'll follow the existing pattern.

Reply:
6d05548